api/v1beta1/helmrepository_types.go

```diff
@@ -72,6 +72,12 @@ type HelmRepositorySpec struct {
 	// AccessFrom defines an Access Control List for allowing cross-namespace references to this object.
 	// +optional
 	AccessFrom *acl.AccessFrom `json:"accessFrom,omitempty"`
+
+	// Type of the HelmRepository.
+	// When this field is set to  "OCI", the URL field value must be prefixed with "oci://".
+	// +kubebuilder:default:="Default"
```

Comment:
```suggestion
	// +kubebuilder:validation:Enum=default,oci
```

Reply:
We use lowercase everywhere else in the APIs, so let's do this there too.

Reply:
I agree. The only reason we used uppercase was that the [RFC](https://github.com/fluxcd/flux2/tree/95bfd3b3a4c59e874678e28cd517f6879b118ed8/rfcs/0002-helm-oci) used mixed-case.